### PR TITLE
feat(clrcore): add new vehicles

### PR DESCRIPTION
**Pull request by Demo4889**
### Goal of this PR
Adding missing vehicles
...

### How is this PR achieving the goal
Adding missing vehicles
...

### This PR applies to the following area(s)
FiveM

### CODE REVIEW

**Comment by ahcenezdh** [comment]
You should rename your pr
"feat(clrcore/enums): add new vehicles"
or smth like that

**Comment by Demo4889** [comment]
> You should rename your pr "feat(clrcore/enums): add new vehicles" or smth like that

Thanks, I was just doing that before you commented lol

**Comment by operationairstrike** [comment]
Yo @Demo4889 it looks like the check has failed I read the logs and it say your title doesnt fit the requirements, I really like this pull request because I've also noticed some vehicles are missing. So could you please fix the title to fit those guidelines in order for it to be pushed into main soon?

Titles need to:
- Start with: fix, tweak, or feat,
- Within parenthesis contain the area the change is applying,
- Colon + space, i.e: ": " (without quotes)
- Description longer than 8 characters".

**Comment by operationairstrike** [comment]
Yo @Demo4889 can you rerun the failed check?

**Comment by Demo4889** [comment]
I'll check it out when I get home later. I'm at work right now.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: code/client/clrcore/External/VehicleHash.cs [hunk 1]
```diff
@@ -118,17 +150,31 @@ public enum VehicleHash : uint
 		Cognoscenti2 = 3690124666u,
 		Comet2 = 3249425686u,
 		Comet3 = 2272483501u,
+		Comet4 = 1561920505u,
+		Comet5 = 661493923u,
+		Comet6 = 2568944644u,
+		conada = 3817135397u
```
[the remark this answers is not in the document]
Comment (Demo4889) [reply code/client/clrcore/External/VehicleHash.cs]: Thank you for noticing that. I wouldn't have even seen that myself. I'm going through the listing again and adding more.

**Review by PsychoShock — COMMENTED** [review]
Looks good for everything, can you now do:

- Run the Github check again (should be good now)
- Squash everything following [the doc](https://docs.fivem.net/docs/contributing/git/squash-guide/)

Good job!

**Comment by Demo4889** [comment]
> Looks good for everything, can you now do:
> 
> * Run the Github check again (should be good now)
> * Squash everything following [the doc](https://docs.fivem.net/docs/contributing/git/squash-guide/)
> 
> Good job!

I've tried checking into running it again. Every time it fails it says it's because of the commit names being the wrong format. I can't even find where to run the check again on my end.

**Comment by AvarianKnight** [comment]
You have to squash down your commit and fix the naming for the CI to run again.

**Pull request closed without merging**